CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 2.11.4
+
+- Fix Bird's Eye Report throwing error when it included archived entities
```

Comment:
Why do we have archived entities in the Bird's eye report?

Reply:
We don't include those in report, only the IDs are in the hierarchy response